API.md

```diff
@@ -0,0 +1,49 @@
+<pre>Activities</pre>
```

Comment:
Actually this will be single activity with meta data which might be part of lesson plans.